config/action.d/ufw.conf

```diff
@@ -13,9 +13,11 @@ actionstop =
 
 actioncheck = 
 
-actionban = [ -n "<application>" ] && app="app <application>" ; ufw insert <insertpos> <blocktype> from <ip> to <destination> $app
+actionban = [ -n "<application>" ] && app="app <application>"
+            ufw insert <insertpos> <blocktype> from <ip> to <destination> $app
 
-actionunban = [ -n "<application>" ] && app="app <application>" ; ufw delete <blocktype> from <ip> to <destination> $app
+actionunban = [ -n "<application>" ] && app="app <application>"
+            ufw delete <blocktype> from <ip> to <destination> $app
```

Comment:
hm, indeed... I wonder how it worked for original submitter since I guess he did test, but app wouldn't be assigned here if in the single line.  so -- thanks!  would you mind adding a ChangeLog entry, THANKS entry and indent for unbacn so ufw under [? ;-)